users/templates/users/activation.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Confirmation</title>
+</head>
+<body>
+<script>
+    alert('Your account has been activated!');
```

Comment:
Better just include the line into body and use the base template so that user could get back to using the site.